src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -2,454 +2,539 @@
  * @jest-environment jsdom
  */
 
-import { library } from '@fortawesome/fontawesome-svg-core'
-import { faClose, faUser } from '@fortawesome/free-solid-svg-icons'
-import { faCoffee, faCircle, faAlien } from '../__fixtures__/icons'
-import { compileAndMount, mountFromProps, coreHasFeature, REFERENCE_ICON_USING_STRING, REFERENCE_ICON_BY_STYLE, ICON_ALIASES } from '../__fixtures__/helpers'
-import FontAwesomeIcon from '../FontAwesomeIcon'
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faClose, faUser } from "@fortawesome/free-solid-svg-icons";
+import { faCoffee, faCircle, faAlien } from "../__fixtures__/icons";
+import {
+  compileAndMount,
+  mountFromProps,
+  coreHasFeature,
+  REFERENCE_ICON_USING_STRING,
+  REFERENCE_ICON_BY_STYLE,
+  ICON_ALIASES,
+} from "../__fixtures__/helpers";
+import FontAwesomeIcon from "../FontAwesomeIcon";
 
 beforeEach(() => {
-  library.add(faCoffee, faCircle, faAlien)
-})
+  library.add(faCoffee, faCircle, faAlien);
+});
 
 afterEach(() => {
-  library.reset()
-})
-
-describe('icons are showing', () => {
-  test('using array format, short prefix and short icon name', () => {
-    const wrapper = mountFromProps({ icon: ['fas', 'coffee'] })
-
-    expect(wrapper.element.tagName).toBe('svg')
-    expect(wrapper.element.classList.contains('fa-coffee')).toBeTruthy()
-  })
+  library.reset();
+});
+
+describe("icon title prop", () => {
```

Comment:
This `describe` (originally, lines 26-55) block are the only tests I added.  
All other changes are prettier.